reports/serializers.py

```diff
@@ -1,8 +1,42 @@
+from typing import List
+
 from reports.models import EnrolmentReport
 from rest_framework import serializers
 
 
+class Size2ArrayFieldBase(serializers.Field):
```

Comment:
Kinda nitpick, but now that this is an actual field that is used, it would probably make sense to remove `Base` from the name.